src/microservices/Microservices.CohortPackager/Execution/ExtractJobStorage/MongoDB/MongoExtractJobStore.cs

```diff
@@ -89,21 +163,10 @@ protected override void PersistMessageToStoreImpl(ExtractedFileStatusMessage mes
 
         protected override void PersistMessageToStoreImpl(ExtractedFileVerificationMessage message, IMessageHeader header)
```

Comment:
As far as I can see, this function isn't called on `ExtractedFileVerificationMessage`s anymore (outside of test cases), instead `AddToWriteQueue` is being called. Is it still being used elsewhere, and if not, do we want to keep it regardless?

Reply:
You're right actually; while I've stated that the batch processing mode is optional, this isn't the case! I'll fix this.